app/src/main/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenter.java

```diff
@@ -2,32 +2,40 @@
 
 import android.support.annotation.NonNull;
 
+import java.util.ArrayList;
 import java.util.List;
 
 import io.intrepid.contest.base.BasePresenter;
 import io.intrepid.contest.base.PresenterConfiguration;
 import io.intrepid.contest.models.Category;
 import io.intrepid.contest.models.Contest;
 
-
-class CategoriesListPresenter extends BasePresenter<CategoriesListContract.View> implements CategoriesListContract.Presenter {
-
+class CategoriesListPresenter extends BasePresenter<CategoriesListContract.View> implements CategoriesListContract.Presenter, CategoryClickListener {
     private final Contest.Builder contestBuilder;
+    private final String categoryName = "Score";
```

Comment:
I think I will still have the same issue of accessing the resources from my Presenter impl if adding this default category locally. Any suggestions on this @gdaniels @jmccormack200